src/scitiff/io.py

```diff
@@ -86,18 +111,55 @@ def export_scitiff(
         _export_data_group(dg, file_path)
 
 
-def load_scitiff(file_path: str | pathlib.Path) -> sc.DataArray | sc.DataGroup:
+def _is_nested_value(
+    model_fields: dict[str, pydantic.fields.FieldInfo], key: str
+) -> bool:
+    return (
+        key in model_fields
+        and (field_type := model_fields[key].annotation) is not None
+        and issubclass(field_type, pydantic.BaseModel)
+    )
+
+
+def _read_image_as_dataarray(
+    image_metadata: ImageDataArrayMetadata, file_path: str | pathlib.Path
+) -> sc.DataArray:
+    image = sc.zeros(
+        dims=[*image_metadata.data.dims],
+        shape=[*image_metadata.data.shape],
+        unit=image_metadata.data.unit,
+        dtype=image_metadata.data.dtype,
+    )
+    tf.imread(file_path, squeeze=False, out=image.values)
```

Comment:
It even fixed the problem with the extra dimension that tifffile adds when loading grayscale image...!